lib/agg_regrid/__init__.py

```diff
@@ -17,7 +17,7 @@
 """A package for experimental regridding functionality."""
 
 from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, xrange, zip)
+from six.moves import (filter, input, map, range, zip)  # noqa
```

Comment:
[noqa](https://flake8.readthedocs.org/en/2.4.1/) is a directive to Flake to prevent it from issuing warnings. We don't want package not used import warnings from Flake when using this simple six import pattern.